src/pydata_sphinx_theme/assets/styles/base/_base.scss

```diff
@@ -185,8 +185,8 @@ pre {
     margin-inline-end: 0.5em;
   }
 
-  @include link-style-hover;
   &:hover {
+    @include link-style-hover;
```

Comment:
Cody tidiness bonus: I like how this change allows everything to be lexically grouped

Reply:
Nice, it's definitely neater than my original approach!